cpu/atmega_common/periph/gpio.c

```diff
@@ -133,7 +135,10 @@ int gpio_init_int(gpio_t pin, gpio_mode_t mode, gpio_flank_t flank,
     uint8_t pin_num = _pin_num(pin);
 
     if ((_port_num(pin) == PORT_D && pin_num > 3)
-         || (_port_num(pin) == PORT_E && pin_num < 4)
+//#if !defined(__AVR_ATmega328P__)
+#if defined (PORTE)
```

Comment:
nope, in avr/libc ports are defined like this PORTA, PORTB, and so on ...
look here for atmega328p definitions : 
http://svn.savannah.nongnu.org/viewvc/trunk/avr-libc/include/avr/iom328p.h?root=avr-libc&view=markup